website/docs/workloads/postgresql/postgresql-profiles.md

```diff
@@ -44,6 +44,34 @@ idea. The name of the client must match the name of the system or the value of t
 }
 ```
 
+## Balanced/In Memory Scenario Support
+In addition to the standard configuration, Virtual Client offers two tuned scenarios to run the Postgresql workload under: Balanced and In-Memory.
+
+* **Balanced**: The database size is about twice as big as the memory/RAM on the system. Half of the database will fit in memory, and half will fit on disk.
+  Target CPU usage is about 40-60% with somewhat heavy disk I/O usage. The configuration supports 1-4 additional data disks, and the database will be
+  distributed among the disks as proportionately as possible.
+* **In-Memory**: The database size is just about the size of the memory/RAM on the system. Target CPU usage is about 80-90%, with a significant amount of disk
+  I/O usage.
+
+A tuned scenario may be selected by denoting it in the PostgreSQLExecutor.
+
+``` bash
+{
+  "Type": "PostgreSQLExecutor",
+  "Parameters": 
+  {
+    "Scenario": "ExecuteTPCCBenchmark",
+    "StressScenario": "Balanced",
```

Comment:
Could you update the parameter section of the postgresql doc to include Balanced?

Reply:
Also add InMemory as well